lib/player.js

```diff
@@ -3596,6 +3596,24 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     return ret;
   }
 
+  /**
+   * Return a copy of the current non default configuration.  Modifications of
+   * the returned value will not affect the Player's active configuration.
+   * You must call <code>player.configure()</code> to make changes.
+   *
+   * @return {!Object}
+   * @export
+   */
+  getNonDefaultConfiguration() {
```

Comment:
You know, this method would have simplified some of the way the demo page handles URI parameters.
Oh well. The system we have now works.